README.md

```diff
@@ -3,6 +3,9 @@ API Common for Java
 
 [![Build Status](https://travis-ci.org/googleapis/api-common-java.svg?branch=main)](https://travis-ci.org/googleapis/api-common-java)
 
+🚌 In January 2023, this library has moved to https://github.com/googleapis/gapic-generator-java/tree/main/api-common-java[gapic-generator-java/api-common-java]. This repository will be archived in the future. Future releases will appear in the new repository (https://github.com/googleapis/gapic-generator-java/releases).
```

Comment:
Markdown link formatting is incorrect

![image](https://user-images.githubusercontent.com/28604/212146986-d9387385-c7fb-4ac9-b7bc-c1e6163dba0e.png)